central/notifier/datastore/datastore_impl.go

```diff
@@ -34,6 +58,23 @@ func verifyNotifierOrigin(ctx context.Context, n *storage.Notifier) error {
 	return nil
 }
 
+func (b *datastoreImpl) GetNotifiersFiltered(ctx context.Context, filter func(notifier *storage.Notifier) bool) ([]*storage.Notifier, error) {
+	if err := sac.VerifyAuthzOK(integrationSAC.ReadAllowed(ctx)); err != nil {
+		return nil, err
+	}
+	notifiers, err := b.storage.GetAll(ctx)
```

Comment:
I remember we might have a ticket for these occurences to change to `store.Walk` once the bolt store is removed.

Can we add a TODO here?